### PR TITLE
Simplified React app generator

### PR DESCRIPTION
**Pull request by Sahana-Srinivasan**
Generates a Hello, Blockstack React app

### CODE REVIEW

**Comment by hstove** [comment]
Still WIP?

**Comment by Sahana-Srinivasan** [comment]
> Still WIP?

Nope, all done, sorry. Forgot to update that comment.

File: react/templates/_package.json [hunk 1]
```diff
@@ -2,41 +2,46 @@
...
-    "url-loader": "^0.5.8",
-    "webpack": "^2.3.3",
-    "webpack-dev-server": "^2.4.2"
-  },
   "dependencies": {
-    "blockstack": "^19.1.0",
-    "bootstrap": "^3.3.7",
-    "react": "^15.5.4",
-    "react-dom": "^15.5.4"
+    "blockstack": "^19.0.1",
```
Comment (zone117x) [inline react/templates/_package.json:15]: Is this version of blockstack.js intentional?

File: react/templates/_package.json [hunk 2]
```diff
@@ -2,41 +2,46 @@
...
+    "bootstrap": "^4.3.1",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
+    "react-router-dom": "^4.3.1",
+    "react-scripts": "3.0.1",
+    "typescript": "^3.5.1"
```
Comment (zone117x) [inline react/templates/_package.json:21]: Typescript dependency shouldn't be needed, are you running into issues without it?

File: react/templates/src/App.js [hunk 3]
```diff
@@ -1,6 +1,7 @@
-import React, { Component, Link } from 'react';
-import Profile from './Profile.jsx';
-import Signin from './Signin.jsx';
+//import React, { Component, Link } from 'react';
```
Comment (zone117x) [inline react/templates/src/App.js:1]: Can we remove this line now?

File: react/templates/src/setupProxy.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+// 'React-scripts start' sets up a development server that automatically
+// registers files with this name. This proxy file sets up a CORS header
+// for manifest.json, allowing sign in via Blockstack without using
+// the webpack configuration file that create-react-app has configured 
+// and hidden.
```
Comment (zone117x) [inline react/templates/src/setupProxy.js:5]: Great comment! A link to their docs about this proxy setup would also be useful.

File: react/templates/src/setupProxy.js [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+module.exports = function(app) {
+  app.get('/manifest.json', (req, res, next) => {
+    res.set({
+      'Access-Control-Allow-Origin': '*',
+      'Access-Control-Allow-Headers': '*',
+      'Access-Control-Allow-Methods': 'GET',
+      'X-random': 'random'
```
Comment (zone117x) [inline react/templates/src/setupProxy.js:13]: Debug header `x-random` can be removed

File: react/templates/src/setupProxy.js (hunk 4 above)
Comment (Sahana-Srinivasan) [reply react/templates/src/setupProxy.js:5]: Will do! I'll update the PR with the other comments incorporated too

**Comment by Sahana-Srinivasan** [comment]
Since this PR's work is encapsulated in #[54](https://github.com/blockstack/blockstack-app-generator/pull/54) I'm going to implement these comments' changes there and close this.

**Pull request closed without merging**